docs/supported-coins.md

```diff
@@ -0,0 +1,24 @@
+# Supported coins
+
+## The pipeline
+
+Do not change `src/data/coins.json` manually.
+
+The one and only source of truth are `*.json` definitions declared and maintained in the [firmware repository](https://github.com/trezor/trezor-firmware/tree/master/common/defs). 
+
+These are duplicated as read-only in the [trezor-common](https://github.com/trezor/trezor-common) repository.
+
+`trezor-common` is included in `trezor-connect` as git submodule mounted at `submodules/trezor-common`.
+
+## Update and maintenance in trezor-connect
+
+_Make sure that desired `[coin].json` definition is present in `trezor-firmware` repository *and* corresponding [support for connect](https://github.com/trezor/trezor-firmware/blob/master/common/defs/support.json#L3) is enabled._
```

Comment:
Small preference to replace https://github.com/trezor/trezor-firmware/blob/master/common/defs/support.json#L3 with https://github.com/trezor/trezor-firmware/blob/4e005de02fbb9db11a304587ec1abe8aab80cdfd/common/defs/support.json#L3 . (You get such link by pressing `y` btw.) This way the link will always link to the same content

Reply:
239c7de